src/libraries/System.Security.Cryptography.Xml/src/System/Security/Cryptography/Xml/CipherData.cs

```diff
@@ -1,15 +1,16 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+using System.Diagnostics.CodeAnalysis;
 using System.Xml;
 
 namespace System.Security.Cryptography.Xml
 {
     public sealed class CipherData
     {
-        private XmlElement _cachedXml;
-        private CipherReference _cipherReference;
-        private byte[] _cipherValue;
+        private XmlElement? _cachedXml;
+        private CipherReference? _cipherReference;
+        private byte[]? _cipherValue;
```

Comment:
since you annotated constructor as non-nullable I'd expect this to be non-null and all related members, or alternatively ctor to accept null but looking at the existing setter I'd assume this is just missing null check in the ctor

Reply:
It's nullable as there are multiple constructors, not all of which set it.